ts/screens/onboarding/OnboardingCompletedScreen.tsx

```diff
@@ -28,38 +22,30 @@ const OnboardingCompletedScreen = () => {
   const idp = O.isSome(idpSelected) ? idpSelected.value.name : "";
   const route = useRoute();
 
-  const continueButtonProps: BlockButtonProps = {
-    bordered: false,
-    title: I18n.t("global.buttons.continue"),
-    onPress: () => {
-      trackLoginEnded(
-        isFastLoginEnabled,
-        idp,
-        getFlowType(false, true),
-        route.name
-      );
-      dispatch(completeOnboarding());
-    }
-  };
-
   useOnFirstRender(() => {
     trackThankYouPageScreen();
   });
 
-  return (
-    <BaseScreenComponent>
-      <SafeAreaView style={IOStyles.flex}>
-        <InfoScreenComponent
-          image={<Pictogram name="fireworks" />}
-          title={I18n.t("onboarding.thankYouPage.title")}
-        />
+  const handleContinue = () => {
+    trackLoginEnded(
+      isFastLoginEnabled,
+      idp,
+      getFlowType(false, true),
+      route.name
+    );
+    dispatch(completeOnboarding());
+  };
 
-        <FooterWithButtons
-          type={"SingleButton"}
-          leftButton={continueButtonProps}
-        />
-      </SafeAreaView>
-    </BaseScreenComponent>
+  return (
+    <OperationResultScreenContent
```

Comment:
Good job in using this already present component. We must just remember that this component doesn't have the automatic a11y focus in its text components. As the focus in the previous page should always be below or approximately around those text components, we expect no side effects, and it's good to go using it. 👍

Reply:
I had forgotten to register that case but it had already been tested and it works! 😁